packages/ember-htmlbars/tests/helpers/yield_test.js

```diff
@@ -1,327 +0,0 @@
-import Ember from 'ember-metal/core';
-import run from 'ember-metal/run_loop';
-import EmberView from 'ember-views/views/view';
-import { computed } from 'ember-metal/computed';
-import { A as emberA } from 'ember-runtime/system/native_array';
-import Component from 'ember-views/components/component';
-import helpers from 'ember-htmlbars/helpers';
-import ComponentLookup from 'ember-views/component_lookup';
-
-import compile from 'ember-template-compiler/system/compile';
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-
-import { registerKeyword, resetKeyword } from 'ember-htmlbars/tests/utils';
-import viewKeyword from 'ember-htmlbars/keywords/view';
-
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var view, owner, originalViewKeyword;
-
-function commonSetup() {
-  owner = buildOwner();
-  owner.registerOptionsForType('template', { instantiate: false });
-  owner.register('component-lookup:main', ComponentLookup);
-}
-
-function commonTeardown() {
-  runDestroy(owner);
-  owner = view = null;
-}
-
-import isEnabled from 'ember-metal/features';
-if (!isEnabled('ember-glimmer')) {
-  // jscs:disable
-
-QUnit.module('ember-htmlbars: Support for {{yield}} helper', {
-  setup() {
-    commonSetup();
-    originalViewKeyword = registerKeyword('view',  viewKeyword);
-  },
-  teardown() {
-    run(function() {
-      Ember.TEMPLATES = {};
-    });
-    runDestroy(view);
-    commonTeardown();
-    resetKeyword('view', originalViewKeyword);
-  }
-});
-
-QUnit.test('a view with a layout set renders its template where the {{yield}} helper appears', function() {
-  var ViewWithLayout = EmberView.extend({
-    layout: compile('<div class="wrapper"><h1>{{attrs.title}}</h1>{{yield}}</div>')
-  });
-
-  view = EmberView.create({
-    withLayout: ViewWithLayout,
-    template: compile('{{#view view.withLayout title="My Fancy Page"}}<div class="page-body">Show something interesting here</div>{{/view}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$('div.wrapper div.page-body').length, 1, 'page-body is embedded within wrapping my-page');
-});
-
-QUnit.test('block should work properly even when templates are not hard-coded', function() {
```

Comment:
@rwjblue can you confirm this test is migrated correctly to "templates should yield to block inside a nested component "? Seems like this might be testing `templateName`, not sure if we still support that (if so, the new test is wrong)